OpenRA.Game/Actor.cs

```diff
@@ -295,6 +295,11 @@ public bool IsDisabled()
 			return false;
 		}
 
+		public bool IsInvisibleTo(Player player)
```

Comment:
I don't like the idea of adding a new method that almost duplicates our existing `Player.CanViewActor`, but inverts everything about it.